kotlin-native/litmus/src/commonMain/kotlin/MemShuffler.kt

```diff
@@ -0,0 +1,31 @@
+import kotlin.properties.ReadWriteProperty
+import kotlin.random.Random
+import kotlin.reflect.KProperty
+
+class MemShuffler(bufferSize: Int, random: Random = Random) : ReadWriteProperty<BasicLitmusTest, Int> {
```

Comment:
Maybe `MemoryShuffler`?
It does not make sense to save few letters here. Moreover in Kotlin code it is preferred to use full nouns in the naming scheme.